modules/goat/taxonsearch/main.nf

```diff
@@ -1,6 +1,6 @@
 process GOAT_TAXONSEARCH {
     tag "$meta.id"
-    label 'process_single'
+    label 'process_low'
```

Comment:
This should remain `process_single`. I know `nf-core lint` complains about this, and I've raised a PR: https://github.com/nf-core/tools/pull/1823

Reply:
I changed it back to `process_single`.